src/core/wopi.py

```diff
@@ -218,6 +218,10 @@ def setLock(fileid, reqheaders, acctok):
                             (op.title(), fn, flask.request.args['access_token'][-20:], e))
 
     try:
+        # If LOCK and oldLock provided this is an UnLockAndRelock operation
```

Comment:
So the idea for now is to revert this part

Reply:
For me that is the crucial part. `UnLockAndRelock` doesn't work currently.

Reply:
Interesting...... can we have a chat over .. gitter? or what else?